### PR TITLE
CellChat

### PR DESCRIPTION
**Pull request by gyear**
For INC1175146 - `CellChat-20210814-foss-2020a-R-4.0.0.eb`

* [x] Assigned to reviewers (usually everyone in apps team)

Default:
* [x] EL8-cascadelake
* [x] EL8-haswell
* [x] Ubuntu20 VM

### CODE REVIEW

File: easybuild/easyconfigs/c/CellChat/CellChat-20210814-foss-2020a-R-4.0.0.eb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    ('cpp11', '0.4.0', {
+        'checksums': ['1768fd07dc30dfbbf8f3fb1a1183947cb7e1dfd909165c4d612a63c163a41e87'],
+    }),
+    ('systemfonts', '1.0.2', {
+        'checksums': ['21ac96412846e06a4062362b159213cb9d83bd60fdf03aa235992b6c49fb36a9'],
+    }),
+    ('svglite', '2.0.0', {
+        'checksums': ['76e625fe172a5b7ce99a67b6d631b037b3f7f0021cfe15f2e15e8851b89defa5'],
+    }),
+    ('RSpectra', '0.16-0', {
```
Comment (rpep) [inline easybuild/easyconfigs/c/CellChat/CellChat-20210814-foss-2020a-R-4.0.0.eb]: We already have this in Bioconductor

Comment (rpep) [reply easybuild/easyconfigs/c/CellChat/CellChat-20210814-foss-2020a-R-4.0.0.eb]: same version too:
https://bear-apps.bham.ac.uk/applications/2020a/R-bundle-Bioconductor/3.11-foss-2020a-R-4.0.0/

File: easybuild/easyconfigs/c/CellChat/CellChat-20210814-foss-2020a-R-4.0.0.eb [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+# Order is important!
+exts_list = [
+    ('NMF', '0.23.0', {
+        'checksums': ['0f0cca01b37bf46fce90d2e951df609d3d377908aa607825083fd0c47cc24753'],
+    }),
+    ('GlobalOptions', '0.1.2', {
+        'checksums': ['47890699668cfa9900a829c51f8a32e02a7a7764ad07cfac972aad66f839753e'],
+    }),
+    ('circlize', '0.4.13', {
```
Comment (rpep) [inline easybuild/easyconfigs/c/CellChat/CellChat-20210814-foss-2020a-R-4.0.0.eb:43]: already in standard R install with 0.4.8, is this newer version a requirement?
https://bear-apps.bham.ac.uk/applications/2020a/R/4.0.0-foss-2020a/

File: easybuild/easyconfigs/c/CellChat/CellChat-20210814-foss-2020a-R-4.0.0.eb [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+
+# Order is important!
+exts_list = [
+    ('NMF', '0.23.0', {
+        'checksums': ['0f0cca01b37bf46fce90d2e951df609d3d377908aa607825083fd0c47cc24753'],
+    }),
+    ('GlobalOptions', '0.1.2', {
```
Comment (rpep) [inline easybuild/easyconfigs/c/CellChat/CellChat-20210814-foss-2020a-R-4.0.0.eb:40]: Already have 0.1.1 in standard R install, do you need newer verison?

Comment (gyear) [reply easybuild/easyconfigs/c/CellChat/CellChat-20210814-foss-2020a-R-4.0.0.eb:40]: circlize 0.4.13 requires, GlobalOptions 0.1.2

**Review by rpep — APPROVED** [review]

**Pull request merged**